src/main/java/com/alexbro/onlinebank/facade/data/factory/DefaultAuthenticationDataFactory.java

```diff
@@ -0,0 +1,22 @@
+package com.alexbro.onlinebank.facade.data.factory;
+
+import com.alexbro.onlinebank.core.entity.user.User;
+import com.alexbro.onlinebank.facade.data.auth.AuthData;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Resource;
+
+@Component
+public class DefaultAuthenticationDataFactory implements AuthenticationDataFactory {
```

Comment:
Rename implementation to 'DefaultAuthDataFactory' and interface to 'AuthDataFactory'